coderd/metricscache/metricscache_test.go

```diff
@@ -150,7 +150,7 @@ func TestCache_BuildTime(t *testing.T) {
 					},
 				},
 				transition: database.WorkspaceTransitionStop,
-			}, want{50 * 1000, true},
```

Comment:
What makes this change needed?

Reply:
The old `dbmem` `tryPercentile` function was implemented slightly differently to how postgres implements `PERCENTILE_CONT` (although this test never gets run with postgres anyway)